src/components/Shared/ButtonWithSpinner/ButtonWithSpinner.tsx

```diff
@@ -23,7 +23,7 @@ const ButtonWithSpinner: FC<Props> = ({
       className={className}
       disabled={disabled || loading}
     >
-      <div className="flex">
```

Comment:
Didn't know this component exists. I guess this could just be move the default button?

Reply:
Usually I Just use that for the confirm Button etc for The loading process

Would it make sense to make It Default?